docs/technical_reference/unit_models/treatment_models/chem_soft.rst

```diff
@@ -0,0 +1,255 @@
+Chemical Softening
+====================================================
+
+This chemical softening unit model calculates the chemical dose required for target removal of hardness causing components. 
+The model also calculates the size of the mixer, flocculator, sedimentation basin and the recarbonation basin. This chemical softening model:
+   * supports steady-state only
+   * predicts the outlet concentration of :math:`\text{Ca}^{2+}` and :math:`\text{Mg}^{2+}`
+   * is verified against literature data
+
+Configuration Inputs
+--------------------
+
+The model requires 2 configuration inputs:
+   * Softening procedure: ``single_stage_lime`` or ``excess_lime`` or ``single_stage_lime_soda`` or ``excess_lime_soda``
+   * Silica removal: ``True`` or ``False``
+
+Degrees of Freedom/Variables
+------------------
+
+The chemical softening model has 18 degrees of freedom that should be fixed for the unit to be fully specified. 
+Additionally, depending on the chemical softening process selected, chemical dosing may or may not be required to be fixed.
+
+Typically, the following 7 variables define the input feed.
+
+.. csv-table::
+   :header: "Variables", "Variable Name", "Symbol", "Unit"
+
+   "Feed volume flow rate", "``properties_in[0].flow_vol_phase['Liq']``", ":math:`Q_{feed}`", ":math:`\text{m}^3 / \text{s}`"
+   "Feed composition Ca2+", "``properties_in[0].flow_mass_phase_comp['Liq','Ca_2+']``", ":math:`m_{Ca^{2+}}`", ":math:`\text{g/}\text{L}`"
+   "Feed composition Mg2+", "``properties_in[0].flow_mass_phase_comp['Liq','Mg_2+']``", ":math:`m_{Mg^{2+}}`", ":math:`\text{g/}\text{L}`"
+   "Feed composition Alkalinity2-", "``properties_in[0].flow_mass_phase_comp['Liq','Alkalinity_2-']``",":math:`m_{alk}`",  ":math:`\text{g/}\text{L}`"
```

Comment:
Seems odd to fix Q and mass flow rates

Reply:
Updated the initial fix to include mass flow rate of H2O instead of volumetric flow rate